src/main/java/net/openhft/chronicle/threads/ThreadHolder.java

```diff
@@ -30,6 +30,11 @@ public interface ThreadHolder {
 
     void resetTimers();
 
+    /**
+     * Get the {@link System#nanoTime()} at which the currently executing loop iteration started
+     *
+     * @return The time the current loop started, or {@link CoreEventLoop#NOT_IN_A_LOOP} if no iteration is executing
+     */
     long startedNS();
```

Comment:
Also clarify where this number comes from, we do arithmetic with it against other timestamps elsewhere. Best to make this explicit.